### DISCUSSION TITLE
input function not working

### DISCUSSION THREAD

**Discussion by ceedee666** [jupyterlite/jupyterlite#762] [Q&A]
Hey everyone,

I have a litte problem using the python `input` function in jupyterlite. When running the jupyterlite debug server using `jupyter lite serve` I can't use the `input` function. What I did is the following. 

- `jupyter lite init`
- `jupyter lite serve`

After that I navigate to `127.0.0.1:8000`and open a python console. When I execute the following simple script:
```python
i = input("Test ")
```
I don't get an imput option. The variable `i` contains the following value:
```python
<Future finished exception=JsException(TypeError: this is null)>
```
However, I don't see any JS exception in the browser console. The only error I can see there is the following:

![image](https://user-images.githubusercontent.com/235757/183600931-7847a1e8-5a16-40c3-8b04-1c493f3b64dd.png)

I'm currently running jupyterlitre on Python 3.9.13 on MacOS. Any idea what the problem could be and how to solve it?

Thanks in advance,
Christian

**Comment by manics** [comment]
This is a known issue: https://github.com/jupyterlite/jupyterlite/issues/275

**Reply by ceedee666** [reply]
Thanks @manics. Somehow I did't find he isse when searching. Will have a look at the discussion there.

**Reply by ceedee666** [reply]
One thing I notived is that the problem does not occur on the [jupyterlite server](https://jupyterlite.readthedocs.io/en/latest/_static/lab/) linked in the documentation. Is the issue just related to the debug server? This isn't clear to me from the discussion in the issue https://github.com/jupyterlite/jupyterlite/issues/275.
